notebook/static/auth/js/loginmain.js

```diff
@@ -5,9 +5,11 @@ define(['jquery', 'base/js/namespace', 'base/js/page'], function($, IPython, pag
     function login_main() {
       var page_instance = new page.Page('div#header', 'div#site');
       $('button#login_submit').addClass("btn btn-default");
+      $('button#login_new_pass_submit').addClass("btn btn-default");
       page_instance.show();
       $('input#password_input').focus();
-    
+      $('input#token_input').focus();
```

Comment:
Focus means putting the cursor in an input so that text will appear there when you start typing. So it doesn't make sense to do it for more than one field.

Reply:
@danagilliann and I will be working on this!